src/templates/ssr/fabric-custom/index.ts

```diff
@@ -23,9 +23,24 @@ const getTag = () => {
 	return Promise.reject('No tag found');
 };
 
+/**
+ * This is a hack to insert the html into the DOM,
+ * using innerHtml would not work if the tag contains a script tag
+ * see https://developer.mozilla.org/en-US/docs/Web/API/Element/innerHTML#security_considerations
```

Comment:
Do the two recommendations: `Element.setHTML()` and `Node.textContent` not work for our use case?

Reply:
Those would sanitise the html/js, so it wouldn't render just appear as text.